src/response/result.ts

```diff
@@ -12,6 +12,7 @@ export interface GuardMatch {
     actualValue: number;   // Using number instead of float
     threshold: string;
     matchCount?: number;    // With default 0
+    matchValues?: string[] | [];
```

Comment:
this doesn't feel like this is necessary. do you need this for the linter or why exactly?